nix/requirements_override.nix

```diff
@@ -21,12 +21,22 @@ let
           )
         );
 
+  cli_common_path =
+    if builtins.pathExists ./../lib/cli_common
+    then ./../lib/cli_common/default.nix
+    else ./../../lib/cli_common/default.nix;
```

Comment:
i think we are now seeing a limitation of using symlinks, since with different levels of folders this is going to be a problem.

i think we will have the same problem with `setup.cfg` which we also symlink.

I don't have a good solution for now. maybe there is a way to use this override file via overlay (then paths are relative to requirements_override.nix file) or we just copy this file into the project and create checks that we ensure proper version of it.